.editorconfig

```diff
@@ -0,0 +1,6 @@
+
+root = true
+
+[*.{js,ts}]
+indent_style = space
+indent_size = 2
```

Comment:
Ok, but that is not editor specific. Is just a pattern that can be used by a lot of editors and that match your current coding style.